site/content/contribute/mobile/build-your-own/assets-white-label.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Assets & White Labeling"
+date: 2018-05-20T11:35:32-04:00
+weight: 3
+subsection: Build Your Own App
+---
+
+We've made it easy to white label the mobile app and to replace override the assets used, however, you have to Build Your Own App from Source\_.
```

Comment:
I think either "replace" or "override" is an extra word here? Also, not sure if "Build Your Own App from Source" should have a link to another doc?

Reply:
it should be replace and override